src/rp2040/hardware_structs/include/hardware/structs/adc.h

```diff
@@ -1,27 +1,86 @@
+// THIS HEADER FILE IS AUTOMATICALLY GENERATED -- DO NOT EDIT
+
 /*
- * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ * Copyright (c) 2021 Raspberry Pi (Trading) Ltd.
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
+
 #ifndef _HARDWARE_STRUCTS_ADC_H
 #define _HARDWARE_STRUCTS_ADC_H
 
 #include "hardware/address_mapped.h"
 #include "hardware/regs/adc.h"
 
+// reference to datasheet: https://datasheets.raspberrypi.org/rp2040/rp2040-datasheet.pdf#tab-registerlist_adc
+// BITMASK : FIELDNAME [BITRANGE] (RESETVALUE): DESCRIPTION
```

Comment:
Looking at the comment-lines below (e.g. `// 0x00007000 [12-14] : AINSEL (0): Select analog mux input`), is it actually
```
// BITMASK  [BITRANGE] : FIELDNAME (RESETVALUE): DESCRIPTION
```
?

Reply:
Ah, whoops. There was a slight change and I forgot to update the comment.

Reply:
Fixed (update soon)